test/config.js

```diff
@@ -0,0 +1,114 @@
+import path from 'path';
+import test from 'ava';
+import sinon from 'sinon';
+import proxyquire from 'proxyquire';
+
+const fixtureBasePath = path.resolve('test', 'fixtures', 'config');
+
+test('should return config only from home-directory when global binary used and `.np-config-json` in home-directory exists', async t => {
+	const homedirStub = sinon.stub();
+	homedirStub.returns(path.resolve(fixtureBasePath, 'homedir1'));
+	const pkgDirStub = sinon.stub();
+	pkgDirStub.onFirstCall().returns(path.resolve(fixtureBasePath, 'pkg-dir'));
+	pkgDirStub.onSecondCall().returns(path.resolve(fixtureBasePath, 'local1'));
+	pkgDirStub.onThirdCall().returns(path.resolve(fixtureBasePath, 'local2'));
+
+	let i = 0;
+	while (i < 3) {
```

Comment:
This test-cases proves, the configuration from home-directory is used no matter what configuration files are provided in the local package directory